config/puma.rb

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env puma
+
+rackup DefaultRackup
+
+environment ENV['RACK_ENV'] || 'development'
```

Comment:
I know it doesn't make a lot of difference by fetch is slightly slower - or at least used to be in ruby 2.2:

https://github.com/JuanitoFatas/fast-ruby#hash-vs-hashfetch-code